src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -1,35 +1,30 @@
 import React, { useState, useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { useRouter } from 'next/router';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
 import {
-  Row, Col, Space, Button, Tooltip, PageHeader, Spin, Collapse,
+  Row, Col, Space, Button, Tooltip, PageHeader, Spin, Collapse, Empty,
 } from 'antd';
-
 import {
   InfoCircleOutlined,
 } from '@ant-design/icons';
 import _ from 'lodash';
 import { Vega } from 'react-vega';
-import useSelection from 'antd/lib/table/hooks/useSelection';
 import plot1Pic from '../../../../../../../static/media/plot9.png';
 import plot2Pic from '../../../../../../../static/media/plot10.png';
-import PlotStyling from '../../filter-cells/components/PlotStyling';
 import CalculationConfig from './CalculationConfig';
 import UMAP from './new_data.json';
```

Comment:
you can remove this file import and also delete the file itself if it is no longer used everywhere